tracer/build/_build/Build.Steps.cs

```diff
@@ -567,6 +568,32 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                     .SetOutput(MonitoringHomeDirectory / framework)));
         });
 
+    Target PublishManagedTracerForAwsLambda => _ => _
+        .Unlisted()
+        .Requires(() => PublishReadyToRun == false || RuntimeIdentifier != null) // RuntimeIdentifier is required if PublishReadyToRun is enabled
+        .After(Clean, Restore)
+        .Executes(() =>
+        {
+            var framework = TargetFramework.NET6_0;     // always target .NET 6 on AWS Lambda
+            var readyToRun = PublishReadyToRun ?? true; // default to ReadyToRun on AWS Lambda unless explicitly disabled
+            var outputDirectory = MonitoringHomeDirectory / framework;
```

Comment:
Thanks. I wasn't sure if we run a separate clean build for this they where they wouldn't share paths.

Reply:
Moved all outputs to the artifacts directory.

Reply:
> Moved all outputs to the artifacts directory.

Thanks!

> I wasn't sure if we run a separate clean build for this they where they wouldn't share paths.

We probably will, but I've fought with this enough locally to predict that this could have caused heartache 😅